Packs/CommonTypes/ReleaseNotes/3_2_47.md

```diff
@@ -0,0 +1,60 @@
+
+#### Layouts
+- **domainRepUnified**
+- **STIX Attack Pattern**
+- **hostRep**
+- **STIX Report**
+##### Intrusion Set
```

Comment:
Should fix the release notes with all those **<>**

Reply:
Done